tests/e2e/cypress/e2e/tabbed-nav.cy.js

```diff
@@ -0,0 +1,124 @@
+    });
+
+    describe("Basic tabbed nav tests", () => {
+      let tabbedNav;
+      it("Can find the tabbed-nav element on the page", () => {
+        tabbedNav = cy.get("tabbed-nav").then((element) => {
+          tabbedNav = element;
+          expect(tabbedNav).to.exist;
+        });
+      });
+
+      it("Knows the tabbed-nav is a defined custom element", () => {
+        cy.window().then((win) => {
+          const customEl = win.customElements.get("tabbed-nav");
+          expect(customEl).to.exist;
+          expect(tabbedNav.get(0).isConnected).to.be.true;
+        });
+      });
+
+      it("There is a default tab selected and its content is visible", () => {
+        cy.get('tabbed-nav .tab-button[aria-expanded="true"]')
+          .as("selectedButton")
+          .should("exist")
+          .get("@selectedButton")
+          .invoke("attr", "data-selected")
+          .should("exist")
+          .get("@selectedButton")
+          .then((btn) => {
+            const tabName = btn.attr("data-tab-name");
+            cy.get(`#${tabName}`).should("exist").should("be.visible");
+          });
+      });
+
+      it("The content of un-selected tabs should not be visible", () => {
+        cy.get(`tabbed-nav .tab-button:not([data-selected])`)
+          .as("unselectedBtn")
+          .should("exist")
+          .get("@unselectedBtn")
+          .then((btn) => {
+            const tabName = btn.attr("data-tab-name");
+            cy.get(`#${tabName}`).should("exist").should("not.be.visible");
+          });
+      });
+
+      it("Clicking an unselected tab should show it", () => {
+        cy.get("tabbed-nav .tab-button:last").as("lastBtn").click();
+        cy.get("@lastBtn").then((btn) => {
+          const tabName = btn.attr("data-tab-name");
+          cy.get(`#${tabName}`)
+            .should("be.visible")
+            .invoke("attr", "data-selected")
+            .should("exist");
+        });
+        cy.get("@lastBtn")
+          .invoke("attr", "data-selected")
+          .should("exist")
+          .get("@lastBtn")
+          .invoke("attr", "aria-expanded")
+          .should("eq", "true");
+      });
+
+      it("Clicking an unselected tab hides the other tabs", () => {
+        cy.get("tabbed-nav .tab-button:last").click();
+        cy.get("tabbed-nav .tab-button:not(:last)").each((btn) => {
+          const tabName = btn.attr("data-tab-name");
+          cy.get(`#${tabName}`)
+            .as("tabContent")
+            .should("not.be.visible")
+            .invoke("attr", "data-selected")
+            .should("not.exist");
+        });
+      });
+    });
+
+    describe("Intercepts click events on above-the-fold alert links", () => {
+      it("Clicking an alert link opens the accordion for that link and scrolls to it", () => {
+        cy.get("weathergov-alert-list a").each((anchorEl) => {
+          const alertId = anchorEl.attr("href").split("#")[1];
+          cy.get(`#${alertId}`).as("alertEl").click();
```

Comment:
This look like it's clicking the accordion element, not the link?

```suggestion
          const alertId = anchorEl.attr("href").split("#")[1];
          cypress.wrap(anchorEl).click();
          cy.get(`#${alertId}`).as("alertEl");
```

Checking my thinking with Cypress, I opened the UI tool so I could rerun just this one test and I noticed that you can expand the test. And then I was surprised you can expand the list of steps, and you can hover over a specific step to see what element it was operating on. Here's the test as-is:

![image](https://github.com/weather-gov/weather.gov/assets/142943695/fdb11024-da6f-4913-973e-81c332a68113)

Which maybe isn't super illustrative. But anyway, when I make that change above, here's what it looks like:

![image](https://github.com/weather-gov/weather.gov/assets/142943695/55827d34-b75a-48bd-a959-dfc534a09a73)

That is so cool! I feel like that's going to be a really useful tool for making sure tests are doing what we intend for them to.

Reply:
I dealt with it in this commit: https://github.com/weather-gov/weather.gov/pull/728/commits/987cf87d82704314343746cdca5206ea5cc6d0a8